packages/openapi-to-graphql/src/schema_builder.ts

```diff
@@ -870,20 +870,48 @@ export function getArgs({
       continue
     }
 
-    // TODO: update with requestOptions
     // If this parameter is provided via options, ignore
     if (typeof data.options === 'object') {
-      if (
-        typeof data.options.headers === 'object' &&
-        parameter.name in data.options.headers
-      ) {
-        continue
-      }
-      if (
-        typeof data.options.qs === 'object' &&
-        parameter.name in data.options.qs
-      ) {
-        continue
+      switch (parameter.in) {
```

Comment:
Before, we were checking both the `qs` and the `headers` options for fulfillment. This is bad. They should have a limited scope.